pkg/setting/setting.go

```diff
@@ -1199,3 +1199,12 @@ func IsExpressionsEnabled() bool {
 	}
 	return v
 }
+
+// IsLiveEnabled returns if grafana live should be enabled
+func IsLiveEnabled() bool {
+	v, ok := FeatureToggles["live"]
+	if !ok {
+		return false
+	}
+	return v
```

Comment:
can you move this to Cfg struct instead and pass this to the Live manager? trying to get rid of global settings variables, somehow FeatureToggles was added to globals 
https://github.com/grafana/grafana/pull/26613